### DISCUSSION TITLE
Upper and Lower Case in Headings

### DISCUSSION THREAD

**Discussion by rainerrose** [razonyang/hugo-theme-bootstrap#498] [Q&A] [answered]
**Describe the bug**
I have noticed that in headings, some words are lowercase while all other words are capitalized.
Personally, I would prefer uniformity. It would be nicer from my point of view if all words were capitalized at the beginning.
I have been able to identify the following words so far:

* in
* or
* and
* the
* a
* an

The words `These`, `Another` and `Other` are not affected.
Unfortunately, I have not found the place where this is accomplished in the source code, otherwise I would patch it myself. 

**Minimal example:**
~~~
+++
title = "caps groß klein in all or and the other these  another a an these schreibung"
date = 2022-03-31T23:17:02+01:00
featured = true
toc = true
draft = true
pinned = true
+++
<!-- Scribers -->
Here
<!--more-->
# foo
we go
~~~

The title would be:

`Caps Groß Klein in All or and the Other These Another a an These Schreibung`

**Expected behavior**
`Caps Groß Klein In All Or And The Other These Another A An These Schreibung`

**Screenshots**
None.

**Hugo version:**

```shell
$ hugo version
hugo v0.96.0-2fd4a7d3d6845e75f8b8ae3a2a7bd91438967bbb+extended linux/amd64 BuildDate=2022-03-26T09:15:58Z VendorInfo=gohugoio
```

**Theme version:**

Installed via Hugo Module 

```shell
go: github.com/razonyang/hugo-theme-bootstrap upgrade => v0.74.0
```

**Suggestion**
Optionally make the behavior parameterizable with a config variable.

**Comment by razonyang** [comment, marked as answer]
In this case, you'll need to use the `Go` as your `titleCaseStyle`(`config.toml`):

```toml
titleCaseStyle = "Go"
```

**Comment by rainerrose** [comment]
Okay, wow @razonyang ! That was really fast. It worked! Thank you very much. I celebrate you for that!
Somehow I didn't suspect the option at Hugo and even with that keyword I would never have found it in this world wide web.
Sorry, maybe this question would have been better placed in the discussions section.
